qa/tasks/thrashosds-health.yaml

```diff
@@ -14,6 +14,7 @@ overrides:
       - \(SMALLER_PGP_NUM\)
       - \(OBJECT_
       - \(SLOW_OPS\)
+      - SLOW_OPS
```

Comment:
```suggestion
      - SLOW_OPS
```

?

Reply:
We also have a warning message of  - SLOW_OPS without brackets, I added it (I don't know why it shows like I also removed on line, checking

Reply:
oh, right